enrollment-server/src/main/resources/db/changelog/changesets/1.5.x/20230302-add-column-fds-data.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
+
+    <changeSet id="enrollment-server-20230302-add-column-fds-data" author="racansky" labels="enrollment-server">
```

Comment:
- [ ] Let's try to use `logicalPath` to avoid problems when refactoring.

Reply:
I suggest depending on the default value and filling it only when refactoring. https://gettingstartedstage.liquibase.com/concepts/changelogs/attributes/logicalfilepath.html

Reply:
Oh, it is useful even if we call liquibase from a different path.